packages/core/src/config/parse.ts

```diff
@@ -1500,12 +1589,22 @@ const parseAndValidateChugSplashConfig = async (
   for (const [referenceName, userContractConfig] of Object.entries(
     userConfig.contracts
   )) {
-    if (
-      userContractConfig.externalProxy !== undefined &&
-      (await provider.getCode(userContractConfig.externalProxy)) === '0x'
-    ) {
+    try {
+      if (
+        userContractConfig.externalProxy !== undefined &&
+        (await provider.getCode(userContractConfig.externalProxy)) === '0x'
+      ) {
+        logValidationError(
+          'error',
+          `Entered a proxy address that does not exist: ${userContractConfig.externalProxy}`,
+          [],
+          cre.silent,
+          cre.stream
+        )
+      }
+    } catch {
```

Comment:
The try/catch seems unnecessary here. We could just check if the proxy's valid via `ethers.utils.isAddress`, then call `logValidationError` if it's invalid